src/main/java/com/o19s/es/ltr/LtrQueryParserPlugin.java

```diff
@@ -171,11 +171,17 @@ public ScriptEngine getScriptEngine(Settings settings, Collection<ScriptContext<
     public List<ActionHandler<? extends ActionRequest, ? extends ActionResponse>> getActions() {
         return unmodifiableList(asList(
                 new ActionHandler<>(FeatureStoreAction.INSTANCE, TransportFeatureStoreAction.class),
+                new ActionHandler<>(FeatureStoreAction.BC_INSTANCE, TransportFeatureStoreAction.BC.class),
```

Comment:
I don't see the corresponding code.
But I'm fine dropping any kind of BC support it since it'll cover only the usecase where an old node communicate to a new one.

Reply:
I am not sure we should worry for now about breaking compatibility, however we should clearly document that. or even probably backport as update for other versions of elasticsearch.

Reply:
I also think that the BC support is too complex to be worthwhile. As david noted this kind of BC support would only allow old node to talk to new one, new talking to old would still fail. Getting around that means needing an intermediate version and 2 cluster restarts.  All for actions that are only administrative and not part of the core data collection/scoring operations.  I'm going to drop this code and add to the documentation somewhere.

Reply:
good, I can also help with documenting and adding test to this tomorrow.